assets/js/analyzer.js

```diff
@@ -143,7 +143,7 @@ function populateSecondaryAnalyzerList() {
 }
 
 function analyze() {
-    if($('#primaryAnalyzerMode').val() === null) {
+    if($('#primaryAnalyzerMode').val() === null || $('#morphAnalyzerInput').val() === '') {
```

Comment:
The best solution is triggering the browser's validation AND not actually making the request.